algebra-utils/src/fft/domain/test.rs

```diff
@@ -42,11 +45,15 @@ fn fft_composition() {
 
     let rng = &mut rand::thread_rng();
 
+    #[cfg(not(feature = "gpu"))]
     test_fft_composition::<MNT6, _>(rng);
+
+    #[cfg(feature = "gpu")]
+    test_fft_composition::<Bls12_381, _>(rng);
 }
 
 #[test]
-fn parallel_fft_consistency() {
+fn fft_consistency() {
```

Comment:
Would be nice to have separate tests for cpu and gpu (even if there will be some code duplication)

Reply:
Done